agriculture/agriculturecore/templates/sidebar.html

```diff
@@ -21,21 +21,21 @@
             </button>
             <div class="dropdown-menu shadow-sm simulation-controls-container" aria-labelledby="simulationMenuButton">
                 <p>Simulation controls</p>
-                <div class="card">
-                    <p class="widget-title">Weather condition</p>
-                    <p class="widget-desc">Control the weather condition of the farm. You can change it to see different temperature and moisture values.</p>
-                    <div class="widget-button-container">
-                        <button id="weather-sunny" class="btn widget-button weather-button" type="button" value="0">
-                            <span id="weather-sunny-logo" class="icon-widget weather-icon fas fa-sun fa-2x" title="Change weather to sunny"></span>
-                        </button>
-                        <button id="weather-cloudy" class="btn widget-button weather-button" type="button" value="1">
-                            <span id="weather-cloudy-logo" class="icon-widget weather-icon fas fa-cloud fa-2x" title="Change weather to cloudy"></span>
-                        </button>
-                        <button id="weather-rainy" class="btn widget-button weather-button" type="button" value="2">
-                            <span id="weather-rainy-logo" class="icon-widget weather-icon fas fa-cloud-rain fa-2x" title="Change weather to rainy"></span>
-                        </button>
-                    </div>
-                </div>
+<!--                <div class="card">-->
```

Comment:
Same as before, just remove this, if we wanted to recover it we can always go back to a previous commit that had this piece of code.